File: comparacao_python/mtd_construtor/Revendedora_1.py
from Automovel import Automovel

class Revendedora_1:
    @staticmethod
    def main():
        novo_automovel = Automovel()
        novo_automovel.mostrar_carro()

        novo_automovel_1 = Automovel(1997,"Volkwagen","Fusca",3888.00)
        novo_automovel_1.mostrar_carro()

if __name__ == "__main__":
    Revendedora_1.main()

File: exec/src/mtd_construtor/Revendedora1.java
package mtd_construtor;

public class Revendedora1 {
    public static void main (String args[]) {
        Automovel A = new Automovel();
        A.mostracarro();

        Automovel B = new Automovel(1997,"Volswagen", "Fusca", 3888.00);
        B.mostracarro();
    }

}
